apps/store/src/features/priceCalculator/PriceCalculatorCmsPage/PriceCalculatorCmsPageContent/PriceCalculatorCmsPageContent.tsx

```diff
@@ -17,9 +18,11 @@ import {
 
 export function PriceCalculatorCmsPageContent() {
   const variant = useResponsiveVariant('lg')
-  const showPurchaseSummary = useAtomValue(priceCalculatorShowPurchaseSummaryAtom)
+  const addedOffer = useAtomValue(priceCalculatorAddedOffer)
```

Comment:
nit: maybe clarify name
```suggestion
  const addedOfferToCart = useAtomValue(priceCalculatorAddedOffer)
```